File: Stage0_Shalini_Gupta_Script.php
<?php
function hamdist($slack,$twitter)
{
	$count=0;
	$i=0;
	while (isset($slack[$i]) != '')
    {
        if ($slack[$i] != $twitter[$i])
            $count++;
        $i++;
    }
	return $count;
}
$name = "Shalini";
$email = "shalinigupta0299@gmail.com";
$slack = "@Shalini";
$biostack = "Data analytics and Vaccine informatics";
$twitter = "@Shashaa";
echo $name," ",$email," ",$slack," ",$biostack," ",$twitter," ";
echo hamdist($slack,$twitter);
?>

File: Stage0_Shalini_Gupta_Script.go
package main
import "fmt"
func hamdist(s1 string,s2 string)int{
  str1 := []rune(s1)
  str2 := []rune(s2)
  var count int
  for i := range str1 {
    if str1[i] != str2[i] {
      count++
		}
	}
	return count
}
func main() {
    var (
        name="Shalini"
        email="shalinigupta0299@gmail.com"
        slack="@Shalini"
        biostack="Data analysts and vaccine informatics"
        twitter="@Shashaa")
    fmt.Println("Name: ",name,"\nEmai: ",email,"\nSlack Id: ",slack,"\nBiostack: ",biostack,"\nTwitter Handle: ",twitter)
    fmt.Println("Hamming Distance: ",hamdist(slack,twitter))
}
